testHelper.js

```diff
@@ -2,6 +2,11 @@ import chai from 'chai';
 import dirtyChai from 'dirty-chai';
 import hook from 'css-modules-require-hook';
 
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
```

Comment:
so this pr adds enzyme too? necessary at this stage?

Reply:
This is enzyme upgrade. It's already used in all tests.